permissions/UDisks.permission

```diff
@@ -1,16 +1,18 @@
 # -*- mode: sh -*-
 
-# FIXME: what exactly requires udisks???
+# x-sailjail-translation-catalog = sailfish-catalog
+# x-sailjail-translation-key-description = permission-la-udisks2
+# x-sailjail-description = Manage storage devices
+# x-sailjail-translation-key-long-description = permission-la-udisks2_description
+# x-sailjail-long-description = Permission to access signals and properties on storage devices, allows to call devices get, lock, mount and format functionality.
```

Comment:
I wouldn't talk about signals or properties here either because those are do not mean anything for user. These descriptions are something user sees. We can describe these separately for developer in README or here as comments.

While this is internal and will not be available in application store, this has translation catalog which means that these will generate translatable strings and translators will translate them. So we should consider that too if we add any texts here. It's also possible to leave that translation catalog empty and it will just fallback to the values here.

Reply:
Right. Lets just keep that empty and mention this only in the internals section.

Reply:
6fbe565b136b84ccbc77dc5d833ed6cdea3663fa ok?